devdoc/CodeReviews.md

```diff
@@ -0,0 +1,55 @@
+Some general guidelines for code reviews.
+
+Code reviews have a few different goals:
+* Catch architectural or design problems
+* Catch bugs early (incorrect behaviour, inefficiencies, security issues)
+* Ensure the code is readable and maintainable.
+* A opportunity for training/passing on information.  
+  For example providing information about how the current system works, functionality that is already available or suggestions of other approaches the developer may not have thought of.
+
```

Comment:
Add a few extra points about PRs here:

- The check boxes are there to remind you to considered that aspect.  If you tick a box and have failed to check it prepare to be commented on!
- The JIRA (or associated documentation) should contain enough details to review the PR - e.g. the purpose/main aim etc..  If the scope of the jira has changed then the jira should be updated to reflect that.
- Something about scanning the code after creating  the PR to check no tracing/other debugging artefacts left in.